scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -140,10 +140,10 @@ else
         xorg-x11-server-Xorg \
         docker-runtime-nvidia \
         oci-add-hooks \
-        libnvidia-container1 \
-        libnvidia-container-tools \
-        nvidia-container-toolkit-base \
-        nvidia-container-toolkit
+        libnvidia-container1-1.16.2-1.x86_64.rpm \
+        libnvidia-container-tools-1.16.2-1.x86_64.rpm \
+        nvidia-container-toolkit-base-1.16.2-1.x86_64.rpm \
+        nvidia-container-toolkit-1.16.2-1.x86_64.rpm
```

Comment:
Would this require rpms to be made available on the packer instance?

Or

I wonder if these need to pinned like `libnvidia-container-1.16.2`, like kepler recipes above

Reply:
you are right, will fix it